openai/client.bal

```diff
@@ -191,12 +193,12 @@ public isolated client class OpenAIClient {
     # Upload a file that contains document(s) to be used across various endpoints/features. Currently, the size of all the files uploaded by one organization can be up to 1 GB. Please contact us if you need to increase the storage limit.
     #
     # + return - OK 
-    resource isolated function post files(CreateFileRequest payload) returns OpenAIFile|error {
+    resource isolated function post files(CreateFileRequest payload) returns json|error {
```

Comment:
This is done to test whether the json response is correct since we had an issue when converting this json to OpenAIFile